src/Esfa.Vacancy.Register.Application/Queries/SearchApprenticeshipVacancies/SearchApprenticeshipVacanciesRequestValidator.cs

```diff
@@ -1,19 +1,27 @@
-﻿using FluentValidation;
+﻿using System.Linq;
+using FluentValidation;
 
 namespace Esfa.Vacancy.Register.Application.Queries.SearchApprenticeshipVacancies
 {
     public class SearchApprenticeshipVacanciesRequestValidator : AbstractValidator<SearchApprenticeshipVacanciesRequest>
     {
+        private const string MinimumFieldsErrorMessage = "At least one of StandardCodes or FrameworkCodes is required.";
         private const int MinimumPageSize = 1;
         private const int MinimumPageNumber = 1;
         private const int MaximumPageSize = 250;
+
         public SearchApprenticeshipVacanciesRequestValidator()
         {
-            RuleFor(r => r.StandardCodes)
-                .NotNull()
-                .WithMessage("At least one search parameter is required.");
+            RuleFor(request => request.StandardCodes)
```

Comment:
Do you not need to have this also for when Frameworks are empty?

Reply:
One rule works for both the cases.